python/ray/experimental/state/custom_types.py

```diff
@@ -1,100 +1,4 @@
-import sys
+from ray.util.state.custom_types import *  # noqa: F401 F403
```

Comment:
- The files are moved to `python/ray/util/state/custom_types.py` 
- `record_deprecated_state_api_import` is called to add warning and telemetry.